test/functional/commands/jetpack-compose-e2e-specs.js

```diff
@@ -0,0 +1,43 @@
+import chai from 'chai';
+import chaiAsPromised from 'chai-as-promised';
+import { initSession, deleteSession, MOCHA_TIMEOUT } from '../helpers/session';
+import { COMPOSE_CAPS } from '../desired';
+
+
+chai.should();
+chai.use(chaiAsPromised);
+
+describe('Jetpack Compose', function () {
+  this.timeout(MOCHA_TIMEOUT);
+
+  let driver;
+  before(async function () {
+    if (process.env.ANDROID_SDK_VERSION === '23') {
```

Comment:
I'd rather check if the SDK version integer number is less or equal than the particular one. Also it would be nice to add a comment here about why the test needs to be skipped

Reply:
Done